01week/rockPaperScissors.js

```diff
@@ -9,9 +9,27 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
-
+  if (hand1 === hand2) {
+    return "It's a tie";
+  } else if (hand1 === 'rock') {
+    if (hand2 === 'scissors') {
+      return 'h1 wins';
+    } else {
+      return 'h2 wins';
+    }
+  } else if (hand1 === 'scissors') {
+    if(hand2 === 'rock') {
+      return 'h2 wins';
+    } else {
+      return 'h1 wins';
+    }
+  } else {
+    if (hand2 === 'scissors') {
+      return 'h2 wins';
+    } else {
+      return 'h1 wins';
+    }
+  }
```

Comment:
Great work. Game is working well and your code looks good. In the future be sure to read and meet the specs, as in: return exactly what the specs say. This will cause you less stress in the future.